docs/docs/explanations/advanced-custom-settings.md

```diff
@@ -134,10 +134,15 @@ jupyterlab:
 ```
 
 :::note Note
-Currently only public git repositories are supported. Path location key should not start or end with trailing slash.
+Path location key should not start or end with trailing slash.
 You can configure JupyterLab to open in a location within the cloned repository by setting `preferred_dir` option within the `jupyterlab` group.
 :::
 
+See also `jupyterlab.gallery_settings` (documented below) which defers the cloning of repositories until user requests it and provides a rich presentation layer.
+
+:::warning
+While you could embed an access token in the URL to fetch from a private repository, please beware that this token be accessed by each user - you should only use tightly scoped personal access tokens which you are comfortable to share with each of your users.
```

Comment:
Typo - this token **will** be accessed by each user

Reply:
```suggestion
While you could embed an access token in the URL to fetch from a private repository, please beware that this token can be accessed by each user - you should only use tightly scoped personal access tokens which you are comfortable to share with each of your users.
```